test/report_test.rb

```diff
@@ -59,18 +60,11 @@ def test_announce_with_options
           text: "• New login\n"
         }
       }
-
-      assert_equal post, SlackDummy.log
```

Comment:
We should probably find a way to preserve this. It's checking that the slack message actually got sent!

Reply:
5036041 🔨